numpydoc/docscrape.py

```diff
@@ -236,10 +236,30 @@ def _parse_param_list(self, content):
 
         return params
 
+    _role = r":(?P<role>\w+):"
+    _funcbacktick = r"`(?P<name>(?:~\w+\.)?[a-zA-Z0-9_.-]+)`"
+    _funcplain = r"(?P<name2>[a-zA-Z0-9_.-]+)"
+    _funcname = r"(" + _role + _funcbacktick + r"|" + _funcplain + r")"
```

Comment:
Are we intentionally disallowing names in backticks not preceded by roles (i.e. using the default role)?

Reply:
That was the effect of the original `_name_rgx`.